### PR TITLE
TTYLIN - Chaosnet TTYLINK stub

### PR DESCRIPTION
**Pull request by larsbrinkhoff**
DEVICE; CHAOS TTYLIN  
BAWDEN; U 14

### CODE REVIEW

**Comment by larsbrinkhoff** [comment]
This server is just a stub which replies "Ouch!  Don't do that!".

**Comment by atsampson** [comment]
There's a client in MINITS - "U - TTYLINK (perhaps Unix only)".

File: doc/programs.md [hunk 1]
```diff
@@ -264,6 +264,7 @@
 - TVWRIT, write TV display contents as bitmap file.
 - TTLOC, advertises physical location of logged in users.
 - TTY, displays TTYs with UNAME/JNAME/CORE and other info.
+- TTYLIN, Chaosnet TTY link.
```
Comment (atsampson) [inline doc/programs.md]: Maybe "Chaosnet TTYLINK server"?

**Comment by larsbrinkhoff** [comment]
Thanks.  Likely the MINITS command "U" is why the ITS server is in a file named "U".

**Comment by larsbrinkhoff** [comment]
Man page for ttylinkd: https://manpages.ubuntu.com/manpages/xenial/man8/ttylinkd.8.html

It's TCP port 87 here: https://www.rfc-editor.org/rfc/rfc1060

**Comment by larsbrinkhoff** [comment]
The Twenex server \<SYS.SYSTEM>TTYLINK.MID.1 is also a stub which responds "Don't insult me, I'm not a UN*X."

**Pull request merged**